examples/pytorch/mnist/Makefile

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
Do we want to keep other 3rd party licences in our examples or we want to remove them ?
What do you think @DeliangFan @kubeflow/wg-training-leads ?

Reply:
I suggest to remove entire bert example from pytorch repo. That PR https://github.com/kubeflow/pytorch-operator/pull/345 brings additional license